src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts

```diff
@@ -38,6 +46,21 @@ export class TerminalChatWidget extends Disposable {
 	private readonly _focusedContextKey: IContextKey<boolean>;
 	private readonly _visibleContextKey: IContextKey<boolean>;
 
+	onDidChangeViewModel!: Event<void>;
+	onDidAcceptInput!: Event<void>;
+	onDidSubmitAgent!: Event<{ agent: IChatAgentData; slashCommand?: IChatAgentCommand }>;
+	onDidChangeAgent!: Event<{ agent: IChatAgentData; slashCommand?: IChatAgentCommand }>;
+	onDidChangeParsedInput!: Event<void>;
+	onDidChangeContext!: Event<{ removed?: IChatRequestVariableEntry[]; added?: IChatRequestVariableEntry[] }>;
+	location!: ChatAgentLocation;
+	viewContext!: IChatWidgetViewContext;
+	viewModel: IChatViewModel | undefined;
+	inputEditor!: ICodeEditor;
+	supportsFileReferences!: boolean;
+	parsedInput!: IParsedChatRequest;
+	lastSelectedAgent: IChatAgentData | undefined;
+	scopedContextKeyService!: IContextKeyService;
```

Comment:
How about instead of this we expose `get chatWidget { return this.inlineChatWidget.chatWidget; }` and get consumers to use that?

Reply:
Had that same thought when I came back after stepping away 😄